src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -94,8 +94,7 @@ public async Task ConfigureAsync(CommandSettings command)
             {
                 // Get the URL
                 var inputUrl = command.GetUrl();
-                if (!inputUrl.Contains("github.com", StringComparison.OrdinalIgnoreCase) &&
-                    !inputUrl.Contains("github.localhost", StringComparison.OrdinalIgnoreCase))
+                if (inputUrl.Contains("codedev.ms", StringComparison.OrdinalIgnoreCase))
```

Comment:
only when it's devfabric, we will request oauth access token, otherwise we always hit github/ghes to request the token via launch app.